packages/decentraland-ecs/src/setupProxy.js

```diff
@@ -0,0 +1,45 @@
+const path = require('path')
+const fs = require('fs')
+
+module.exports = function (dcl, app, express) {
+    const dclKernelPath = path.resolve(dcl.getWorkingDir(), 'node_modules', 'decentraland-kernel');
+    const dclKernelDefaultProfilePath = path.resolve(dclKernelPath, 'default-profile');
+    const dclUnityRenderer = path.resolve(dclKernelPath, 'unity-renderer');
+
+    const routeMappingPath = {
+        '/': {
+            path: path.resolve(dclKernelPath, 'preview.html'), 
+            type: 'text/html'
+        },
+        '/@/artifacts/preview.js': {
+            path: path.resolve(dclKernelPath, 'dist', 'preview.js'), 
+            type: 'text/javascript'
+        },
+        '/@/artifacts/unity-renderer/index.js': {
+            path: path.resolve(dclUnityRenderer, 'index.js'), 
+            type: 'text/javascript'
+        },
+        '/@/artifacts/unity-renderer/unity.data.unityweb': {
+            path: path.resolve(dclUnityRenderer, 'unity.data.unityweb'), 
+            type: 'text/plain'
+        },
+        '/@/artifacts/unity-renderer/unity.framework.js.unityweb':{
+            path: path.resolve(dclUnityRenderer, 'unity.framework.js.unityweb'), 
+            type: 'text/javascript'
+        },
+        '/@/artifacts/unity-renderer/unity.wasm.unityweb':{
+            path: path.resolve(dclUnityRenderer, 'unity.wasm.unityweb'), 
+            type: 'application/wasm'
+        },
```

Comment:
May it make sense to use wildcards? `/@/artifacts/unity-renderer/*` files change across versions, being too specific may not be a wise future-proof desicion.

Example code: https://github.com/decentraland/explorer-website/blob/main/src/setupProxy.js#L13

Reply:
Yes, I'll change it. In the previous commit there wasn't 1-to-1 mapping but now makes sense.